osfoffline/tasks/operations.py

```diff
@@ -132,6 +169,30 @@ def __init__(self, remote):
     @asyncio.coroutine
     def run(self):
         print("Delete Remote File: {}".format(self.remote))
+        db_parent = session.query(models.File).filter(models.File.osf_id == self.remote.parent.id).one()
+        path = os.path.join(db_parent.path, self.remote.name)
+
+        resp = yield from self.remote.request_session.request('DELETE', self.remote.id)
+        yield from resp.release()
+
+         # get item
+        try:
+            item = _get_item_by_path(path)
+        except ItemNotInDB:
+            print('Exception caught: Tried to delete item {}, but it was not found in DB'.format(self.local.name))
```

Comment:
There are separate `DatabaseDeleteFile` and `DatabaseDeleteFolder` events- can we reuse / implement that code, rather than duplicating database operations here?

(`db_task = DatabaseDeleteFile(...) ; db_task.run()`)